README.rst

```diff
@@ -58,6 +58,62 @@ running a version lower than 2.0.
 
      ckan.harvest.mq.type = redis
 
+7. If you want your ckan harvest logs to be exposed to the CKAN API you need to properly 
+   configure the logger. The default configuration logs everything in the database with 
+   log level ``DEBUG``. If you want to modify the default logging mechanism set the following 
+   parameter in your configuration file::
+
+     ckan.harvest.log_scope = 0
+
+ * -1 - Do not log in the database
+ *  0 - Log everything - DEFAULT
+ *  1 - model, logic.action, logic.validators, harvesters
+ *  2 - model, logic.action, logic.validators
+ *  3 - model, logic.action
+ *  4 - logic.action
+ *  5 - model
+ *  6 - plugin
+ *  7 - harvesters
+
+Additionally you can configure the logger the following way::
```

Comment:
@amercader Actually it is an additional configuration option not a requirement so maybe it would be better if I remove it from the README and use it only for the tests?